src/commands/index.rs

```diff
@@ -83,6 +83,13 @@ pub fn add_document(key: &str, index_name: &str, doc: &RedisJSON) -> RedisResult
     REDIS_OK
 }
 
+pub fn remove_document(key: &str, index_name: &str) -> RedisResult {
+    if let Some(schema) = schema_map::as_ref().get(index_name) {
```

Comment:
Please use the explicit `let map = schema_map::as_ref();` as in the other methods for consistency (and I also think it's more readable).

Reply:
:+1: